packages/blade/src/components/Button/BaseButton/StyledBaseButton.d.ts

```diff
@@ -0,0 +1,24 @@
+import type { BaseButtonProps } from './BaseButton';
+import type { ButtonMinHeight } from './buttonTokens';
+
+export type StyledBaseButtonProps = Omit<BaseButtonProps, 'icon' | 'children'> & {
+  activeBorderColor: string;
+  activeColor: string;
+  borderColor: string;
```

Comment:
since we are prefixing all other states then let's prefix this to `defaultBorderColor` to keep it consistent and avoid confusion?